test-e2e/test/helpers.js

```diff
@@ -1,7 +1,5 @@
 // @flow
 
-jest.setTimeout(20000);
```

Comment:
I wanted to use the genFixture function in setup and `jest` is not defiend there.